### ISSUE TITLE
Add Desktop Notification for Blocked Scripts

### ISSUE THREAD

**Issue by selivan** [undergroundwires/privacy.sexy#264]
### Description

Privacy cleanup/Clear Windows logs & caches creates script blocked by Windows Defender: `Trojan:PowerShell/MpTamperPShell.HB`

### OS

Windows 11 22H2 build 22621.2361
privacy.sexy v0.12.4

### Reproduction steps

- check [x] Privacy cleanup/Clear Windows logs
- try to run the script - nothing happens
- try to save the script - nothing happens
- copy script and paste it manually into .bat file - Windows Defender blocks it

### Scripts

https://gist.github.com/selivan/8f390c995dcb282a49f89d26b185274b

### Screenshots

![image](https://github.com/undergroundwires/privacy.sexy/assets/1208989/f5c865d0-6c47-4e8f-ad79-aae044157aff)

**Comment by undergroundwires** [comment]
Hi, thank you for the report.

I could not reproduce this on W10 22H2, even when I copy pasted your gist manually and after triggering a scan manually.

| Windows 10 22H2 | Windows 11 22H2 |
| -------------------------- | --------------------------  |
| ![image](https://github.com/undergroundwires/privacy.sexy/assets/15555035/bd53352e-b602-401a-9f9c-9155ec957caa) | ![image](https://github.com/undergroundwires/privacy.sexy/assets/15555035/106160fb-8f62-44d2-9807-a4b814303e29) |

I can guess that this happens because of `Clear Defender scan (protection) history`. This is most likely because of the fix cb21a970b6b867e1476a5eb8a72b9a7fdd53a744 in [v0.12.4](https://github.com/undergroundwires/privacy.sexy/releases/tag/0.12.4) to solve #246, privacy.sexy now aggressively deletes scan artifacts with high privileges.

I'm not sure how to solve this 🤔

**Comment by selivan** [comment]
@undergroundwires it could be good to add some instructions for user how to ignore it. Notification on a script blocked by Windows Defender, at least. Now it looks like the program just does nothing, very confusing to user.

**Comment by undergroundwires** [comment]
Good idea. It makes sense. But then it will be desktop-app only functionality right? We can not get it working on browser.

**Comment by selivan** [comment]
Of course, for desktop only.

**Comment by undergroundwires** [comment]
I will implement this, thank you for the idea again ♥️

**Comment by undergroundwires** [comment]
I suddenly start getting this when I run the command by copy pasting.

```
At line:1 char:1
+ $command = 'del "C:\ProgramData\Microsoft\Windows Defender\Scans\Hist ...
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
This script contains malicious content and has been blocked by your antivirus software.
    + CategoryInfo          : ParserError: (:) [], ParentContainsErrorRecordException
    + FullyQualifiedErrorId : ScriptContainedMaliciousContent
```

I realize that we do not even need to run this script as TrustedInstaller but can just take ownership of the files before deleting them. I will change the implementation of this so it does not get flagged like this.

**Comment by undergroundwires** [comment]
I don't know what happened but privacy.sexy suddenly cannot run any scripts anymore, not only the ones removed by Defender. I tested with `0.12.4` and `0.12.5`. Could you let me know if you can run other scripts if you're around @selivan ?

**Comment by selivan** [comment]
@undergroundwires 

privacy.sexy 0.12.5
Windows 11 22H2, all updates installed.

Just successfully ran "Disable OS data collection/Disable Windows telemetry and data collection/Disable error reporting".

**Comment by undergroundwires** [comment]
I will add support for showing system dialogs to solve this. It will also enable functionality where we can prompt for creating a restore point before running the scripts as requested in #50.

**Comment by undergroundwires** [comment]
Update: Several minor issues affecting script execution reliability have been resolved. The primary cause of the script execution failure I experienced was due to privacy.sexy app and its generated script executing as administrator by default on Windows which led to it running in the background without any visible window. This is fixed will be included in an upcoming patch release.  I'm also working on integrating an informative dialog to alert users when script execution fails, including due to missing file. Hopefully I can make it to next release.

Meanwhile I'm looking for feedback for a long term solution in #304.
